docs/ABOUT.md

```diff
@@ -0,0 +1,5 @@
+_The ECMAScript specification is a standardized specification of a scripting language developed by Brendan Eich of Netscape; initially it was named Mocha, later LiveScript, and finally JavaScript._ --[Wikipedia](https://en.wikipedia.org/wiki/ECMAScript)
```

Comment:
There are two occurrences of *specification*. If the paragraph is identical to the source (Wikipedia) there's no problem, but I would start the paragraph this way: *ECMAScript is a standardized specification...*